docs/docs/reference/release-notes/v3.8/index.md

```diff
@@ -0,0 +1,46 @@
+---
+date: "2021-06-22"
+version: "3.8.0"
+title: "v3.8 Release Notes"
+---
+
+# [v3.8](https://github.com/gatsbyjs/gatsby/compare/gatsby@3.8.0-next.0...gatsby@3.8.0) (June 2021 #2)
+
+Welcome to `gatsby@3.8.0` release (June 2021 #2)
+
+Key highlights of this release:
+
+- [Web Vitals Tracking](#web-vitals-tracking) - Analytics Plugins now support tracking Web Vitals
+- [name2](#link2) - one-line description
+
+Also check out [notable bugfixes](#notable-bugfixes--improvements).
+
+**Bleeding Edge:** Want to try new features as soon as possible? Install `gatsby@next` and let us know
+if you have any [issues](https://github.com/gatsbyjs/gatsby/issues).
+
+[Previous release notes](/docs/reference/release-notes/v3.7)
+
+[Full changelog](https://github.com/gatsbyjs/gatsby/compare/gatsby@3.8.0-next.0...gatsby@3.8.0)
+
+---
+
+## Web Vitals Tracking
+
+By using the [`web-vitals`](https://github.com/GoogleChrome/web-vitals) package we've added support for tracking Web Vitals to `gatsby-plugin-google-analytics` & `gatsby-plugin-google-tagmanager` ([PR #31665](https://github.com/gatsbyjs/gatsby/pull/31665)). They send three metrics:
+
+- **Largest Contentful Paint (LCP)**: measures loading performance. To provide a good user experience, LCP should occur within 2.5 seconds of when the page first starts loading.
+- **First Input Delay (FID)**: measures interactivity. To provide a good user experience, pages should have a FID of 100 milliseconds or less.
+- **Cumulative Layout Shift (CLS)**: measures visual stability. To provide a good user experience, pages should maintain a CLS of 1 or less.
```

Comment:
```suggestion
- **Cumulative Layout Shift (CLS)**: measures visual stability. To provide a good user experience, pages should maintain a CLS of 0.1 or less.
```

Reply:
https://web.dev/cls/

Reply:
@calcsam that's not true, in analytics it needs to be integered so that's why it's 1 here